roles/cdi/tasks/provision.yml

```diff
@@ -15,48 +15,29 @@
   when: ns.rc != 0
         and platform == "openshift"
 
+- name: Render CDI operator
+  template:
+    src: "cdi-operator.yaml.j2"
+    dest: "/tmp/cdi-operator.yaml"
+
+- name: Create CDI Operator
+  command: "{{ cluster_command }} apply -f /tmp/cdi-operator.yaml --validate=false"
+
 - name: Render {{ cdi_namespace }} ResourceQuota deployment yaml
```

Comment:
@mhenriks Is the ResourceQuota still needed?

Reply:
I had no idea it existed, and neither did anyone else in the storage standup.   So I say you can remove it.

Reply:
done